client/src/components/ReviewsDialog/ReviewsDialog.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Dialog, DialogTitle, List } from '@mui/material';
+import Review from '../../interface/Review';
+import { useTheme } from '@mui/material';
+import ProfileReview from '../ProfileReview/ProfileReview';
+
+interface Props {
+  profileName: string;
+  open: boolean;
+  onClose: () => void;
+  reviews: Review[];
+}
+
+export default function ReviewsDialog({ profileName, open, onClose, reviews }: Props): JSX.Element {
+  const theme = useTheme();
+
+  return (
+    <Dialog fullWidth={true} maxWidth="sm" onClose={onClose} open={open}>
+      <DialogTitle>Reviews for {profileName}</DialogTitle>
+      <List>
+        {reviews.map((review, i) => {
+          return (
+            <Box sx={{ padding: `${theme.spacing(2)} ${theme.spacing(2)}` }} key={i}>
```

Comment:
The usage of indexes as keys is an anti-pattern in React, please use unique keys instead